python/paddle/fluid/dygraph/container.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .layers import Layer
+
+__all__ = ['Sequential']
+
+
+class Sequential(Layer):
+    """Sequential container.
+    Sub layers will be added to this container in the order of argument in the constructor.
+    The argument passed to the constructor can be iterable Layers or iterable name Layer pairs.
+
+    Parameters:
+        name_scope(str): The name of this class.
+        layers(iterable): Iterable Layers or iterable name Layer pairs.
+
+    Examples:
+        .. code-block:: python
+
+            import paddle.fluid as fluid
+            import numpy as np
+
+            data = np.random.uniform(-1, 1, [30, 10]).astype('float32')
+            with fluid.dygraph.guard():
+                data = fluid.dygraph.to_variable(data)
+
+                # create Sequential with iterable Layers
+                model1 = fluid.dygraph.Sequential('model1',
+                    fluid.FC('fc1', 2),
+                    fluid.FC('fc2', 3)
+                )
+                model1[0]  # access fc1 layer
+                res1 = model1(data)  # sequential execution
+
+                # create Sequential with name Layer pairs
+                model2 = fluid.dygraph.Sequential('model2',
+                    ('l1', fluid.FC('l1', 2)),
+                    ('l2', fluid.FC('l2', 3))
+                )
+                model2['l1']  # access l1 layer
+                model2.add_sublayer('l3', fluid.FC('l3', 3))  # add sublayer
```

Comment:
add some comments to show where `l3` will be added

Reply:
done